components/plan-build/bin/hab-plan-build.sh

```diff
@@ -846,9 +847,9 @@ _attach_whereami() {
   echo "From: $src @ line $lnum :"
   echo
   awk '{printf "%d: %s\n", NR, $0}' "$src" \
-    | sed -e "$((${lnum}-${context})),$((${lnum}+${context}))!d" \
+    | sed -e "$((lnum - context)),$((lnum + context))!d" \
```

Comment:
See https://github.com/koalaman/shellcheck/wiki/SC2004#-is-unnecessary-on-arithmetic-variables